react/training/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
+
+import LoginForm from './components/layout';
+
+class Login extends Component {
+  submit = values => {
+    return <Redirect to={{ pathname: '/' }} />;
```

Comment:
This is not working. Here you have to change your own state and then redirect to the `Game`.
for example:
```
class Login extends Component {
    state = {
        isLoggedIn: false
    };

    handleSubmit = values => {
        this.setState({ isLoggedIn: true });
    };

    render() {
        return !this.state.isLoggedIn ? <LoginForm onSubmit={this.handleSubmit} /> : <Redirect to="/game" />;
    }
}
export default Login;
```